tools/automation/tests/__init__.py

```diff
@@ -17,6 +17,11 @@
 
 IS_WINDOWS = sys.platform.lower() in ['windows', 'win32']
 TEST_INDEX_FORMAT = 'testIndex_{}.json'
+PROFILE_TO_NAMESPACE = {
+    '2017-03-09-profile': 'profile_2017_03_09',
+    '2018-03-01-hybrid': 'hybrid_2018_03_01',
+    'latest': 'latest'
+}
```

Comment:
This is just one more thing to maintain.  These follow a consistent pattern, why not just special-case latest and for the others, do the dash to underscore replacement and move the end word to the beginning.